### PR TITLE
trans docs/troubleshooting.md

### PR DESCRIPTION
**Pull request by chuanhaojin**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

trans docs/troubleshooting.md

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/troubleshooting_zh.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+2. 如果使用 `nohup` 启动 cloudcore/edgecore，会默认在日志所在的位置添加一个路径，如果没有提供日志文件，则会将日志写入stdout。
+
+## 我们在哪里找到pod日志
+
+连接到边缘节点，然后
+
+- 查看位于`/var/log/pods` 中的日志文件 或者
+- 使用如同 `docker logs <container id>` 的命令来操作
+
+KubeEdge尚不支持**kubectl logs**。
```
[the remark this answers is not in the document]
Comment (chuanhaojin) [reply docs/troubleshooting_zh.md:68]: 好的，英文版修改后一并提交。

File: docs/troubleshooting_zh.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+然后你可以在没有任何网络问题的情况下进行安装，`keadm` 会检测到它们，并且不会再次下载(请确保您指定`v1.4.0`作为您的版本`--kubeedge-version 1.4.0`).
+
+## 容器一直处于挂起/终止状态
+
+1. 检查 `kubectl get nodes` 的输出，确认该节点是否运行正常。检查“kubectl get nodes”的输出，该节点是否运行正常。请注意，处于无法访问、脱机状态的节点在恢复正常之前无法完成非侵害式的pod删除操作的删除操作。
+2. 检查 `kubectl describe pod <your-pod>`的输出，确认pod是否调度成功。
+3. 检查 `edgecore` 日志是否有任何错误。
+4. 检查运行 `edgecore` 的节点的体系结构，并确保尝试运行的容器映像具有相同的体系结构。
+
+   For example, if you are running `edgecore` on Raspberry Pi 4, which is of `arm64v8` architecture, the nginx image to be executed would be `arm64v8/nginx` from the docker hub.
```
[the remark this answers is not in the document]
Comment (chuanhaojin) [reply docs/troubleshooting_zh.md]: 再次感谢您review本次PR所花的时间。@fisherxu 

已将上述问题修复重新发起PR了

后续翻译的时候会更加仔细和严格。

**Comment by fisherxu** [comment]
/lgtm
Please squash the commits, then let's get ot merged.

**Comment by fisherxu** [comment]
This pr also include some unrelated files.

**Pull request closed without merging**